apps/web/src/components/Feed/Socialize/AdmireButton.tsx

```diff
@@ -47,137 +58,16 @@ export function AdmireButton({ eventRef, queryRef }: AdmireButtonProps) {
     `,
     queryRef
   );
-
-  const [admire] = usePromisifiedMutation<AdmireButtonMutation>(graphql`
-    mutation AdmireButtonMutation($eventId: DBID!, $connections: [ID!]!) @raw_response_type {
-      admireFeedEvent(feedEventId: $eventId) {
-        ... on AdmireFeedEventPayload {
-          __typename
-          feedEvent {
-            viewerAdmire
-              @appendNode(edgeTypeName: "FeedEventAdmireEdge", connections: $connections) {
-              dbid
-              ...AdmireNoteFragment
-            }
-          }
-        }
-        ... on ErrAdmireAlreadyExists {
-          __typename
-        }
-      }
-    }
-  `);
-
-  const [removeAdmire] = usePromisifiedMutation<AdmireButtonRemoveMutation>(graphql`
-    mutation AdmireButtonRemoveMutation($admireId: DBID!) @raw_response_type {
-      removeAdmire(admireId: $admireId) {
-        ... on RemoveAdmirePayload {
-          __typename
-          admireID
-        }
-
-        ... on ErrAdmireNotFound {
-          __typename
-        }
-      }
-    }
-  `);
-
-  const reportError = useReportError();
-  const { pushToast } = useToastActions();
   const { showModal } = useModalActions();
-  const track = useTrack();
 
-  const interactionsConnection = ConnectionHandler.getConnectionID(
-    event.id,
-    'Interactions_admires'
-  );
-  const admireModalConnection = ConnectionHandler.getConnectionID(
-    event.id,
-    'AdmiresModal_interactions'
-  );
+  const track = useTrack();
 
   const handleRemoveAdmire = useCallback(async () => {
```

Comment:
moved out all of the mutation logic into hooks , `useAdmirePost` and `useAdmireFeedEvent`